test/service/test_manifest.py

```diff
@@ -648,481 +637,460 @@ def test_bdbag_manifest_for_redundant_entries(self):
+            ('specimen_from_organism.organ.ontology_label', 'pancreas', 'pancreas', '', ''),
+            ('specimen_from_organism.organ.text', 'pancreas', 'pancreas', 'brain||tumor', 'brain||tumor'),
+            ('specimen_from_organism.organ_part.ontology', 'UBERON:0000006', 'UBERON:0000006', '', ''),
+            ('specimen_from_organism.organ_part.ontology_label', 'islet of Langerhans', 'islet of Langerhans', '',
+             ''),
+            ('specimen_from_organism.organ_part.text', 'islet of Langerhans', 'islet of Langerhans', '', ''),
+
+            ('specimen_from_organism.provenance.document_id',
+             'a21dc760-a500-4236-bcff-da34a0e873d2',
+             'a21dc760-a500-4236-bcff-da34a0e873d2',
+             'aaaaaaaa-7bab-44ba-a81d-3d8cb3873244||b4e55fe1-7bab-44ba-a81d-3d8cb3873244',
+             'aaaaaaaa-7bab-44ba-a81d-3d8cb3873244||b4e55fe1-7bab-44ba-a81d-3d8cb3873244')
+        ]
+        self._assert_tsv(expected, response)
+
+    @manifest_decorator
     def test_full_metadata_missing_fields(self):
         self.maxDiff = None
         bundle_fqid = BundleFQID('f79257a7-dfc6-46d6-ae00-ba4b25313c10', '2018-09-14T133314.453337Z')
         self._index_canned_bundle(bundle_fqid)
-        # moto will mock the requests.get call so we can't hit localhost; add_passthru let's us hit the server
-        # see this GitHub issue and comment: https://github.com/spulec/moto/issues/1026#issuecomment-380054270
-        with ResponsesHelper() as helper:
-            helper.add_passthru(self.base_url)
-            storage_service = StorageService()
-            storage_service.create_bucket()
 
-            filters = {'project': {'is': ['Single of human pancreas']}}
+        fieldnames = []
+        for filters in ({'project': {'is': ['Single of human pancreas']}},
+                        {'project': {'is': ['Mouse Melanoma']}}):
             response = self._get_manifest(ManifestFormat.full, filters)
             self.assertEqual(200, response.status_code, 'Unable to download manifest')
             # Cannot use response.iter_lines() because of https://github.com/psf/requests/issues/3980
             lines = response.content.decode('utf-8').splitlines()
             tsv_file1 = csv.reader(lines, delimiter='\t')
-            fieldnames1 = set(next(tsv_file1))
-
-            filters = {'project': {'is': ['Mouse Melanoma']}}
-            response = self._get_manifest(ManifestFormat.full, filters)
-            self.assertEqual(200, response.status_code, 'Unable to download manifest')
-            # Cannot use response.iter_lines() because of https://github.com/psf/requests/issues/3980
-            lines = response.content.decode('utf-8').splitlines()
-            tsv_file2 = csv.reader(lines, delimiter='\t')
-            fieldnames2 = set(next(tsv_file2))
+            fieldnames.append(set(next(tsv_file1)))
+        intersection = reduce(operator.and_, fieldnames)
+        symmetric_diff = reduce(operator.xor, fieldnames)
```

Comment:
```suggestion
        symmetric_diff = reduce(set.intersection, fieldnames)
```

Reply:
Assuming you meant `set.symmetric_difference` here.